pkg/test/framework/settings/flags.go

```diff
@@ -18,62 +18,41 @@ import (
 	"flag"
 	"fmt"
 	"os"
-	"strings"
-
-	"github.com/spf13/cobra"
-	"github.com/spf13/pflag"
 
 	"istio.io/istio/pkg/log"
 )
 
-var settings = defaultSettings()
-var showHelp = false
-var logOptions = log.DefaultOptions()
-
-// The command we use to parse flags.
-var cmdForLog = &cobra.Command{}
+var (
+	settings = defaultSettings()
+	// TODO(nmittler): Add logging options to flags.
```

Comment:
@nmittler Do we need to add option `OutputPaths`,`ErrorOutputPaths` to flags? We should use Cobra flags for those slice var?